lib/pages/chat/chat.dart

```diff
@@ -2001,7 +2020,6 @@ class ChatController extends State<Chat>
     pinnedMessageScrollController.dispose();
     onUpdateEventStreamSubcription?.cancel();
     keyboardVisibilitySubscription?.cancel();
-    InViewNotifierListCustom.of(context)?.dispose();
```

Comment:
why we remove it?

Reply:
In the case of `InViewNotifierListCustom`, it is a widget that manages its own state and resources internally. You do not need to call dispose on it because Flutter's framework will handle the disposal of its resources when the widget is removed from the tree.